src/dune/dune_load.ml

```diff
@@ -224,15 +225,27 @@ let interpret ~dir ~project ~(dune_file : File_tree.Dune_file.t) =
 
 let load ~ancestor_vcs =
   File_tree.init ~ancestor_vcs ~recognize_jbuilder_projects:false;
-  let projects =
+
+  let _, vcs, projects =
```

Comment:
The code below is not entirely trivial. Please add some comments about how `ancestor_vcs` and `vcs` interact.

Reply:
I have documented `ancestor_vcs` in the public API - my understanding of it at least, it was present and I just took it in account.